README.md

```diff
@@ -1,59 +1,74 @@
-# \<link[rel="lazy-import"]\> [![Build Status](https://img.shields.io/travis/Polymer/lazy-imports.svg?style=flat-square)](https://travis-ci.org/Polymer/lazy-imports)
+# \<link rel="lazy-import"\> 
 
-Declarative lazy HTML imports
+[![Build Status](https://img.shields.io/travis/Polymer/lazy-imports.svg?style=flat-square)](https://travis-ci.org/Polymer/lazy-imports)
+
+This repo implements declarative, lazy, HTML Imports. Normal HTML Imports are eager, meaning that they are loaded and evaluated in order first, before any code that follows. There is also a [javascript API](https://www.polymer-project.org/1.0/docs/devguide/instance-methods#imports-and-urls) for performing HTML Imports, allowing lazy loading of code at any point in time. Lazy loading lets you build an app in a sharded way, so that only the minimal amount of code is loaded to display the current view. This is a key piece of [the PRPL pattern](https://developers.google.com/web/fundamentals/performance/prpl-pattern/).
```

Comment:
+1, done.

The link doesn't work right now, but it looks like it'll be fixed on the next deploy of the docs sight.